main/perl-inline/APKBUILD

```diff
@@ -17,12 +17,12 @@ builddir="$srcdir"/$_pkgname-$pkgver
 
 prepare() {
 	default_prepare
-	export CFLAGS=`perl -MConfig -E 'say $Config{ccflags}'`
+	export CFLAGS="$(perl -MConfig -E 'say $Config{ccflags}')"
```

Comment:
Currently `apkbuild-cpan` does not include quotation marks in the new templates. Perhaps use it to regenerate this? Or update also it to contain the quotes. In any case, expansion like this should not strictly need the quotes, and that's why I omitted them. Thoughts?